projects/core/src/lib/layers/layer-list/layer-list-element/layer-list-element.component.scss

```diff
@@ -52,10 +52,6 @@ mat-progress-bar {
     padding: 0 !important;
 }
 
```

Comment:
Layer name and layer menu font size not equal
![image](https://user-images.githubusercontent.com/1027102/210561184-7aad4acf-c786-46d2-87ad-d7514c2956a2.png)

Reply:
I guess its more that the layer menu manually sets a random font size that is not n'sync with the fon size that is used by a component. I changed it… but we should rethink this for future refactorings.

[89ff6e5](https://github.com/geo-engine/geoengine-ui/pull/555/commits/89ff6e588663cc51144cc98fafb5711ae0330b65)